src/zino/trapd/__init__.py

```diff
@@ -1,6 +1,29 @@
 """Zino SNMP trap daemon back-ends"""
 
-# This is where we switch between back-ends, for now
-# from zino.trapd.pysnmp_backend import *  # noqa
+import importlib
+from types import ModuleType
 
-from zino.trapd.netsnmpy_backend import *  # noqa
+_selected_backend = None
+_BACKEND_MAP = {"pysnmp": "pysnmp_backend", "netsnmp": "netsnmpy_backend", None: "netsnmpy_backend"}
```

Comment:
These are identical in src/zino/snmp/__init__.py. Why two places?

Reply:
Because the implementations in `zino.snmp` and `zino.trapd` are essentially independent of each other.  The sub-modules just happen to be named the same under both `zino.snmp` and `zino.trapd`, but this way, the modules under `zino.trapd` could be renamed to `foobar` and `stinkyface`, and it would not matter to the `zino.snmp` namespace, only `zino.trapd._BACKEND_MAP` would need to change.